project/src/main/webapp/index.html

```diff
@@ -8,7 +8,14 @@
   </head>
   <body>
     <div class="section">
-      <div id="hamburger-icon"><div></div></div>
+      <div id="hamburger-icon" onclick="toggleNavBar()"><div></div></div>     
```

Comment:
What does the additional <div></div> do?

Reply:
the inner div was to make the 3 lines of the nav icon, and the outer div was to add hover effects to all 3 lines at once, instead of individually. I can try a different method if that'd be preferable?

Reply:
This method is fine by me, it was more curiosity :)
also woah the "< div > < / div > in my original comment just like added line breaks to my comment